ui/src/websites/wordpress/WordPressAccountInfo.tsx

```diff
@@ -0,0 +1,80 @@
+import { Button, Form, Input, message, Radio } from 'antd';
+import _ from 'lodash';
+import React from 'react';
+import { WordPressAccountData } from 'postybirb-commons';
+import LoginService from '../../services/login.service';
+import { LoginDialogProps } from '../interfaces/website.interface';
+
+export default class WordPressAccountInfo extends React.Component<
+  LoginDialogProps,
+  WordPressAccountData
+> {
+  state: WordPressAccountData = {
+    username: '',
+    app_password: '',
+    instance: '',
+  };
+
+  constructor(props: LoginDialogProps) {
+    super(props);
+    this.state = {
+      ...this.state,
+      ...(props.data as WordPressAccountData)
+    };
+  }
+
+  async submit() {
+    LoginService.setAccountData(this.props.account._id, this.state)
+      .then(() => {
+        // TODO: somehow check if login info even has perms to post to the instance
+        message.success('WordPress login info saved');
+      })
+      .catch(() => {
+        message.error('Failed to save login info');
+      });
+  }
+
+  render() {
+    return (
+      <div className="container mt-6">
+        <Form layout="vertical">
+          <Form.Item
+            label="Username"
+            help="Your WordPress username."
+          >
+            <Input
+              className="w-full"
+              defaultValue={this.state.username}
+              onBlur={({ target }) => this.setState({ username: target.value })}
+            />
+          </Form.Item>
+          <Form.Item
+            label="App Password"
+            help="Your WordPress application password."
+          >
+            <Input
```

Comment:
nit: at a minimum this should at least be a password type input as to not show plaintext passwords